components/apim-appgw/local.tf

```diff
@@ -1,10 +1,10 @@
 locals {
-  env = var.env == "perftest" ? "test" : var.env
+  env = var.env == "perftest" ? "test" : var.env == "aat" ? "stg" : var.env
 
-  vnet_rg     = var.env == "ithc" || var.env == "perftest" || var.env == "demo" ? "hmcts-hub-nonprodi" : "hmcts-hub-${var.env}-int"
-  vnet_name   = var.env == "ithc" || var.env == "perftest" || var.env == "demo" ? "hmcts-hub-nonprodi" : "hmcts-hub-${var.env}-int"
+  vnet_rg     = var.env == "ithc" || var.env == "perftest" || var.env == "demo" ? "hmcts-hub-nonprodi" : var.env == "aat" ? "hmcts-hub-prod-int" : "hmcts-hub-${var.env}-int"
```

Comment:
Dont think line 4/5 are needed? if not in env, it will default to prod hub

Reply:
I think this is needed as the default is "hmcts-hub-${var.env}-int" and _hmcts-hub-aat-int_ is invalid. I'm guessing its this way to cover sbox as well